prow/config/jobs/istio.yaml

```diff
@@ -76,6 +76,20 @@ jobs:
       - name: INTEGRATION_TEST_FLAGS
         value: --istio.test.skipVM
 
+  - name: integ-ambient-calico
```

Comment:
```suggestion
  - name: integ-ambient-calico
    modifiers: [presubmit_optional]
```

I think it should be optional until we know it's working as expected, then we can make it required.

Reply:
Sounds like a good plan. TY